### PR TITLE
Religion Fixes

### LINKED ISSUE #682: Nazen Religion changes
**Describe your suggestion in full detail below:**
_This has been split off from a larger issue_
Checked out Nerubians one day and some things didn't seem right about their religion
- Move one Nazeni holy site from Nordrassil to Ulduar. I honestly didn't even get this one. Nerubians never did ANYTHING in Kalimdor or Nordrassil, they don't worship nature nor they're full-on magically power hungry. In that matter Nerubians did try to overtake Ulduar, so it will make way more sense than Nordrassil.
- Disallow bastards(doesn't make sense for them to have bastards, since adultery is accepted and they're technically still a swarm)
- Disallow marriage of priests(viziers didn't needed to reproduce until after War of the Spider)
- Clerical function recruitment(we can't add a new religious tenet(for now maybe) and viziers did fight(Warcraft 3, WoW), so it would only make sense)
- Make priests only male(only females in Nerubian society were queens and they don't seem like religious leaders to me).

### PR DESCRIPTION
**Pull request by Loralius**
## Changelog:
**Reworks**
- Jinyu Waterspeaker Faith Rework: New tenets and doctines in-line with [lore flavour.](https://wowpedia.fandom.com/wiki/Jinyu#Culture)
- Doctrine **Chi Pacifism** now confers mutual opinion of _astray_.
  - Applies to Waterspeaker and Pandaren Religion.

**Doctrines**
- Arcane is now shunned by Amani. 
- Pandaren and Jinyu ignore life magic.
- Scarlet Faith accepts fire and arcane magic.
- Nazen considers no adherents bastards.
- Nazen clerical function (taxation -> recruitment)
- Nazen clerical gender (either -> male)

**Holy Sites**
- Ula-Tek and Karazhan now grant positive opinion from rulers of different faiths. 
- Stratholme replaces Ironforge holy site for **Holy Light** faith. 
- Nazen Holy Sites replaced (Nordrassil, Wintergarde) -> (Ulduar, Coldarra)

### ***Completed***

- [x] Searched for useless tidemother faith triggers using totalcommander
- [x] Closes #682 

**Waterspeaker Rework**
<details>
<summary> Click to show details </summary> 

_Tenets_
- Monasticism -> Religious Law
- Life Syncretism -> Esotericism

_Religious Attitude and Clerical Tradition_
- Lay Clergy -> Theocratic
- Fundamentalist -> Righteous
- Spiritual for Life

_Marriage_
- Concubines -> Monogamy 
- Divorce Allowed -> Divorce Approval 

_Crimes (From default Water Group Accepted)_
- homosexuality_shunned
- adultery_men_shunned
- adultery_women_shunned
- kinslaying_extended_family_crime
- deviancy_crime
- witchcraft_crime

</details>

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [X] The mod takes less than 5.5 GB in the Task Manager (Windows)

### CODE REVIEW

**Comment by Loralius** [comment]
Jinyu still has life syncreticism as a doctrine. Any ideas for a good replacement?

**Comment by Grekopithikos** [comment]
I think this may be relevant to this #827 except for the last part maybe

**Comment by Grekopithikos** [comment]
> I think this may be relevant to this #827 except for the last part maybe

Well maybe not

**Comment by Loralius** [comment]
> > I think this may be relevant to this #827 except for the last part maybe
> 
> Well maybe not

I will include the Ironforge->Stratholme change for Holy Light on this branch.

**Comment by Loralius** [comment]
I'm mulling over adding the Celestial Communion doctrine to Hozen as well, seeing as they were uplifted and seemingly [worshipped the August Celestials in the past](https://wowpedia.fandom.com/wiki/Hozen)[^1] (do they still in the present?). That being said, it likely forms a small or ancillary part of Apism and probably not enough for Apism as a whole to be considered only astray by Pandaren and Jinyu.

[^1]: See '1.5 history'

File: common/religion/doctrines/wc_doctrines_special.txt [hunk 1]
```diff
@@ -212,3 +212,17 @@ doctrine_legions_puppet = {
...
+		has_doctrine = special_doctrine_celestial_communion # Keeps on creating a new faith
+	}
+	
+	special_doctrine_celestial_communion = {
+		parameters = {
+			special_doctrine_celestial_communion_active = yes 
+			hostility_override_special_doctrine_celestial_communion = 1
+		}
+	}
+}
```
Comment (zumbak04) [inline common/religion/doctrines/wc_doctrines_special.txt:228]: Maybe make the existing tenet of Pacifism work the same way?
<details>
<summary>Click to expand</summary>

image

</details>

File: common/religion/religions/water_group.txt [hunk 2]
```diff
@@ -278,16 +278,35 @@
 			holy_site = yulon
 			holy_site = xuen
 			
+			doctrine = special_doctrine_celestial_communion
+			
 			#Main Group
 			doctrine = doctrine_no_head
+			doctrine = doctrine_pluralism_righteous
```
Comment (zumbak04) [inline common/religion/religions/water_group.txt:285]: Maybe make them and Celestial pluralistic?

Comment (Loralius) [reply common/religion/religions/water_group.txt:280]: I can see it either way but am leaning towards righteous for both. Pandaren don't strike me as a particularly bigoted people but they are very insular and arguably conservative in traditions, this is already reflected in the magic doctrines where everything except elemental practice is ignored or illegal. The Jinyu are more firmly in the righteous camp given their structured caste based society which is practically built around the cult of waterspeakers.

**Comment by MemeWizard** [comment]
dfb7f62 closes #764, and also edits trigger to accommodate transferring of the pandaren religion to the elemental religious family
@Loralius check that one, please

**Comment by Grekopithikos** [comment]
It stroke me as odd that Masons can't legitimize bastards when other Light faiths can #856
Unless there is some lore reason I wasn't aware of

File: localization/english/religion/wc_core_tenets_l_english.yml [hunk 3]
```diff
@@ -121,4 +121,6 @@
  tenet_bloody_feast_desc:0 "The most valuable sacrifice we can offer is the sacrifice of sentient life. Blood goes to the divine while the flesh can be consumed to absorb the wisdom and strength of the deceased."
 
  special_doctrine_legions_puppet_name:0 "$game_concept_legions_puppet$"
- special_doctrine_legions_puppet_desc:0 "Our master [obey_legion|E] for now and thus we follow, but nothing can withstand death."
\ No newline at end of file
+ special_doctrine_legions_puppet_desc:0 "Our master [obey_legion|E] for now and thus we follow, but nothing can withstand death."
+ 
+ doctrine_parameter_hostility_override_tenet_dharmic_pacifism:0 "Considers and is considered by [faiths|E] with Chi Pacifism to be [GetHostilityLevelName( FAITH_DOCTRINE_PARAMETER.GetIntValue )|V]"
```
Comment (MemeWizard) [inline localization/english/religion/wc_core_tenets_l_english.yml:126]: To have same localization pattern as in gnostic(vanilla)/eastern(mod) hostility overrides
```suggestion
 doctrine_parameter_hostility_override_tenet_dharmic_pacifism:0 "Considers other Chi Pacifistic [faiths|E] to be [GetHostilityLevelName( FAITH_DOCTRINE_PARAMETER.GetIntValue )|V]"
```

**Review by MemeWizard — APPROVED** [review]

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**